### ISSUE TITLE
[Deprecated Package]: Microsoft.EntityFrameworkCore.SqlServer.Design

### ISSUE THREAD

**Issue by mfkoca** [dotnet/core#8233]
### Package Id

Microsoft.EntityFrameworkCore.SqlServer.Design

### Package Version(s)

1.1.6

### Additional details

I'm wondering why it was removed and I don't know which one should I use instead of this package?

**Comment by liveans** [comment]
I think, this package has been deprecated in favor of `Microsoft.EntityFrameworkCore.Design`, but I think, @ajcvickers can provide more accurate answer.

**Comment by ajcvickers** [comment]
Correct. The code in the package was included into the main provider package for newer versions of EF Core. This package should not be used.
